examples/complicated/validators.go

```diff
@@ -228,5 +228,8 @@ func (r User) Validate() error {
 			}
 		}
 	}
+	for _, x := range r.ByteArray {
```

Comment:
for now array does not know about underlying validation, it was achieved by meaningless generation